vendor/bat-native-ads/src/bat/ads/ad_content_info_unittest.cc

```diff
@@ -15,8 +15,6 @@ namespace ads {
 class BatAdsAdContentInfoTest : public UnitTestBase {
  protected:
   BatAdsAdContentInfoTest() = default;
```

Comment:
Constructor is also not needed, so it can removed. So there will be following class definition:

`class BatAdsAdContentInfoTest : public UnitTestBase {};`

Reply:
Changed in this PR rather than a followup. Thanks